pkgs/top-level/all-packages.nix

```diff
@@ -35813,22 +35813,20 @@ with pkgs;
 
   wapiti = callPackage ../tools/security/wapiti { };
 
-  wayfireApplications = wayfireApplications-unwrapped.withPlugins (plugins: [ plugins.wf-shell ]);
-  inherit (wayfireApplications) wayfire wcm;
-  wayfireApplications-unwrapped = recurseIntoAttrs (
-    (callPackage ../applications/window-managers/wayfire/applications.nix { }).
-    extend (_: _: { wlroots = wlroots_0_16; })
-  );
-  wayfirePlugins = recurseIntoAttrs (
-    callPackage ../applications/window-managers/wayfire/plugins.nix {
-      inherit (wayfireApplications-unwrapped) wayfire;
-    }
-  );
-
   weave-gitops = callPackage ../applications/networking/cluster/weave-gitops { };
 
+  wayfire = callPackage ../applications/window-managers/wayfire/default.nix {
+    wlroots = wlroots_0_16;
+  };
   wf-config = callPackage ../applications/window-managers/wayfire/wf-config.nix { };
 
+  wayfirePlugins = recurseIntoAttrs (
```

Comment:
I would prefer `wayfire.plugins`.

Reply:
We have defined the `wayfire` package

 > error: attribute 'wayfire.plugins' already defined at /home/rewine/nur/nixpkgs/pkgs/top-level/all-packages.nix:35818:3

Maybe we can rename `wayfire `to `wayfire.default`, but that is strange

Reply:
What??  O.O
Let's ignore this for now